Chisel/Makefile

```diff
@@ -6,5 +6,6 @@ install:
 		-configuration Release \
 		-sdk iphonesimulator \
 		install \
+		LD_DYLIB_INSTALL_NAME="$(LD_DYLIB_INSTALL_NAME)" \
```

Comment:
@keith Sir Make, how do I conditionally do this? If there's no `LD_DYLIB_INSTALL_NAME`, then I want to leave this out, so that it remains unset.